yew/src/virtual_dom/vtag.rs

```diff
@@ -130,27 +126,29 @@ impl VTag {
         }
     }
 
-    /// Adds a single class to this virtual node. Actually it will set by
-    /// [Element.setAttribute](https://developer.mozilla.org/en-US/docs/Web/API/Element/setAttribute)
-    /// call later.
-    pub fn add_class(&mut self, class: &str) {
-        self.classes.push(class);
-    }
-
-    /// Adds multiple classes to this virtual node. Actually it will set by
-    /// [Element.setAttribute](https://developer.mozilla.org/en-US/docs/Web/API/Element/setAttribute)
-    /// call later.
-    pub fn add_classes(&mut self, classes: Vec<&str>) {
-        for class in classes {
-            self.classes.push(class);
+    /// List of attached classes.
+    pub fn classes(&self) -> &str {
+        if let Some(class_str) = self.attributes.get("class") {
+            class_str.as_ref()
+        } else {
+            ""
         }
     }
 
     /// Add classes to this virtual node. Actually it will set by
     /// [Element.setAttribute](https://developer.mozilla.org/en-US/docs/Web/API/Element/setAttribute)
     /// call later.
+    /// Or if the classes are empty it will be removed by
+    /// [Element.removeAttribute](https://developer.mozilla.org/en-US/docs/Web/API/Element/removeAttribute)
+    /// call later.
     pub fn set_classes(&mut self, classes: impl Into<Classes>) {
```

Comment:
Maybe we should remove this as well. I don't think it's needed anymore, we can call `add_attribute` directly

Reply:
`set_classes` is just convenient, but not needed.
It just makes sure, that the attribute is not set to "" if the Classes are empty, but this logic could be moved to the macro.